pkg/activator/certificate/cache.go

```diff
@@ -40,23 +41,26 @@ import (
 type CertCache struct {
 	secretInformer v1.SecretInformer
 	logger         *zap.SugaredLogger
+	trust          netcfg.Trust
```

Comment:
Maybe: trustConfig? Its not clear from context that this is a configuration flag.